app/generator/content/statement/part3/get-table/index.js

```diff
@@ -15,12 +15,12 @@ const getTable = (fundings) => {
         [
           { text: 'Standard', style: 'tableHeader' },
           { text: 'Level', style: 'tableHeader' },
-          { text: 'Rate', style: 'tableHeader' },
+          { text: 'Rate (£)', style: 'tableHeader' },
```

Comment:
Does this need the additional text in the ticket including as it was never clarified - rate per ha is included on the layout we are following, but not included in the code.